docs/guides/testing/testing.rst

```diff
@@ -642,28 +399,6 @@ Tool(s) available for evaluating complexity of edx-platform code:
 Other Testing Tips
 ==================
 
-Connecting to Browser
----------------------
-
-If you want to see the browser being automated for JavaScript or bok-choy tests,
```

Comment:
Does this still apply to `JavaScript`? I imagine it will apply to some sort of testing in the future.